index.html

```diff
@@ -571,18 +571,13 @@
       <p>
         The <a>PushManager</a> interface defines the operations to access <a>push services</a>.
       </p>
-      <dl title="interface PushManager" class="idl">
-        <dt>
-          Promise&lt;PushSubscription&gt; subscribe (optional PushSubscriptionOptionsInit options)
-        </dt>
-        <dt>
-          Promise&lt;PushSubscription?&gt; getSubscription ()
-        </dt>
-        <dt>
-          Promise&lt;PushPermissionState&gt; permissionState (optional PushSubscriptionOptionsInit
-          options)
-        </dt>
-      </dl>
+      <pre class="idl">
+interface PushManager {
+    Promise&lt;PushSubscription&gt; subscribe(optional PushSubscriptionOptionsInit options);
+    Promise&lt;PushSubscription?&gt; getSubscription();
+    Promise&lt;PushPermissionState&gt; permissionState(optional PushSubscriptionOptionsInit options);
+};
+</pre>
       <p>
         The <code><dfn id=
         "widl-PushManager-subscribe-Promise-PushSubscription--PushSubscriptionOptionsInit-options">subscribe</dfn></code>
```

Comment:
Here and elsewhere: the names of the members [aren't being linkified](https://cdn.rawgit.com/martinthomson/push-api/contiguous-webidl/index.html#extensions-to-the-serviceworkerregistration-interface) anymore. It looks like the definitions of the members need to indicate that they are, so this `<code>...</code>` should change to:

`<dfn for="PushManager" title="subscribe"><code>subscribe</code></dfn>`

(modulo #58, so in whichever order you prefer)